### ISSUE TITLE
Image preprocessing

### ISSUE THREAD

**Issue by KrSiddhartha** [ultralytics/yolov5#3399]
## ❔Question
So i am trying to work on pneumonia detection for a kaggle challenge ( https://www.kaggle.com/c/rsna-pneumonia-detection-challenge). The image size provided is 1024x1024 with single channel. I converted it to 3 channels and normalised and centred the bboxes. Do I need to do any other preprocessing ?

I understand the model handles normalisation but the training batch image seems off with the bounding boxes

## Additional context
Image and bounding boxes as they appear in training batch

![train_batch1](https://user-images.githubusercontent.com/26413292/120250136-b4436580-c29a-11eb-8045-50935dde0520.jpg)

Image and bounding box for a single x-ray
![__results___24_0](https://user-images.githubusercontent.com/26413292/120250269-1f8d3780-c29b-11eb-8ba3-d0174a9d1835.png)

**Comment by glenn-jocher** [comment]
@KrSiddhartha single-channel images load automatically as 3-ch and train/detect correctly, you shouldn't need to do any modifications to your images.

Regarding your labels, if they look incorrect to you then they are incorrect and you should fix them.

**Comment by glenn-jocher** [comment]
@KrSiddhartha see Train Custom Data Tutorial:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
